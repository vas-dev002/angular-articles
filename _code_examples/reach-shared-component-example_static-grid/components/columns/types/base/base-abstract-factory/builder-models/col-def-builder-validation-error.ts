export class ColDefBuilderValidationError extends Error { }
