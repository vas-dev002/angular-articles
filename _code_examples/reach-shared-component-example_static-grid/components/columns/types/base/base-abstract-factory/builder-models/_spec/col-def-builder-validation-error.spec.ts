import { ColDefBuilderValidationError } from "../col-def-builder-validation-error";

declare const expect: jest.Expect;

describe("[Model]: ColDefBuilderValidationError", () => {
	let colDefBuilderValidationError: ColDefBuilderValidationError;

	beforeEach(() => {
		colDefBuilderValidationError = new ColDefBuilderValidationError();
	});

	describe("[ColDefBuilderValidationError instantiate]", () => {
		it("should be successfully created", () => {
			expect(colDefBuilderValidationError).toBeTruthy();
			expect(colDefBuilderValidationError).toMatchSnapshot();
		});
	});
});
