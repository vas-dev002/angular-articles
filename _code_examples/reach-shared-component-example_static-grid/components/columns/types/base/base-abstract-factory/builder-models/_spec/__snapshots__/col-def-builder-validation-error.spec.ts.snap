// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Model]: ColDefBuilderValidationError [ColDefBuilderValidationError instantiate] should be successfully created 1`] = `[Error]`;
